app/src/main/java/com/yterletskyi/happyfriend/features/friends/data/GlobalFriends.kt

```diff
@@ -6,4 +6,9 @@ object GlobalFriends {
         contactId = -1L,
         position = -10L,
     )
+    val MyGlobalIdea = Friend(
+        id = "my_wishlist",
+        contactId = -1L,
```

Comment:
I'm not sure `contactId = -1L` is going to work. If I remember it correctly there cannot be multiple friends with the same contact id (-1L). So you better change it to `-2L`

Reply:
aha, setting `contactId = -1L` means that "general ideas" item will point to the same contact that "my wishlist" does. So ideas will be the same for both.